### PR TITLE
[12.0][IMP] web_widget_x2many_2d_matrix: center header cell

### PR DESCRIPTION
**Pull request by alexey-pelykh**
Minor

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Odoo doesn't center header columns by default AFAIK.

**Comment by alexey-pelykh** [comment]
EE grid does, though, and compare this:
image
and
image

**Comment by pedrobaeza** [comment]
Well, it depends the set of data. Any way, for me is OK.

**Review by pedrobaeza — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
Yet as usual - I'm open to suggestions how to do that in best way possible

**Comment by alexey-pelykh** [comment]
@tarteo @simahawk if you don't mind, please check on this as well

**Comment by alexey-pelykh** [comment]
@Yajo any thoughts on this?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: web_widget_x2many_2d_matrix/static/src/scss/web_widget_x2many_2d_matrix.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.o_field_x2many_2d_matrix .row-total {
+    font-weight: bold;
+}
+  
+.o_x2many_2d_matrix {
+    thead > tr > th {
+        white-space: pre-line;
```
Comment (yajo) [inline web_widget_x2many_2d_matrix/static/src/scss/web_widget_x2many_2d_matrix.scss:8]: You should insert a comment here to explain your use case, which is quite specific AFAICS, if you don't want it to disappear in the next migration by an innocent migrator.

**Review by yajo — APPROVED** [review]

File: web_widget_x2many_2d_matrix/static/src/scss/web_widget_x2many_2d_matrix.scss (hunk 1 above)
Comment (alexey-pelykh) [reply web_widget_x2many_2d_matrix/static/src/scss/web_widget_x2many_2d_matrix.scss:8]: The list has its headers processed in the same way, I'm not even sure how to document it. "It should always have been like that"? :)

Comment (yajo) [reply web_widget_x2many_2d_matrix/static/src/scss/web_widget_x2many_2d_matrix.scss:8]: Mmm maybe "Respect newline characters in headers".

File: web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
             var $table = $('<table>').addClass(
                 'o_list_view table table-condensed table-striped'
             );
+            $table.addClass('o_x2many_2d_matrix')
```
Comment (tarteo) [inline web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js]: Why not put this class in the addClass on the previous line?

Comment (alexey-pelykh) [reply web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js]: Frankly speaking, I don't know. 80-line formatting I guess. How to better format that?

Comment (yajo) [reply web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js]: joining 2 strings is a solution.

Comment (tarteo) [reply web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js]:
```
            var $table = $('<table>').addClass(
                'o_list_view table table-condensed table-striped' +
                'o_x2many_2d_matrix'
            );
```
This is fine (for me though)

Comment (alexey-pelykh) [reply web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js]: Done

**Review by tarteo — APPROVED** [review]
Thanks looks good!

**Pull request merged**